app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::V1::ReviewsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def create
+    new_review = Review.new(review_params)
+    new_review.user = current_user
+
+    if (new_review.rating != "" && new_review.rating != nil)
+      new_review.rating = new_review.rating.to_i
+    end
```

Comment:
Im not sure exactly what this check is doing but pretty sure that this should be removed. 

Your model level and db level constraints check to ensure it is an integer that is being receiving. if your validations are preventing the persistence of a good review, then you are handling something strange on the front end before we get to this create method. Your fetch POST should always contain an integer for rating because a rating is required